extensions/html-language-features/client/src/requests.ts

```diff
@@ -20,34 +22,34 @@ export namespace FsReadDirRequest {
 
 export function serveFileSystemRequests(client: CommonLanguageClient, runtime: Runtime, subscriptions: { dispose(): any }[]) {
 	subscriptions.push(client.onRequest(FsContentRequest.type, (param: { uri: string; encoding?: string; }) => {
-		const uri = Uri.parse(param.uri);
-		if (uri.scheme === 'file') {
+		const uri = param.uri.match(uriScheme);
```

Comment:
why did you change this?

Reply:
Based on earlier comments I thought `Uri.parse` shall explicitly NOT be used to check for schemes.
I actually actually liked it better before but it feels a bit inconsistent to use sometimes use a bare regex and sometimes use `Uri.parse`. Don't have a string opinion here. I can change it back. Lat me know what you prefer.

One more important thing:
The `serveFileSystemRequests(..)` helper was already in the code, I did not create it.
However, it was unused - so someone had written that helper but never activated it (or deactivated it for some reason).
I just activated added in [client.onReady()](https://github.com/microsoft/vscode/pull/137557/files#diff-326f44a4671f469344f799a8c570e485b446e6b32fc74c9265bc2a5662295fe2R123) in htmlClient.ts (same way it is done in css-language-features where pretty much the same code has been active all along).
Was there a reason why these handlers were added but commented out? Seems a bit suspicious..

Reply:
uti is always in the URI format. So Uri.parse is ok to use.

The point is to no use `Uri.parse` on a relative path.